pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/containerd/component_test.go

```diff
@@ -126,11 +126,9 @@ containerd_monitoring
 
 	logRotateData = `/var/log/pods/*/*/*.log {
     rotate 14
-    copytruncate
```

Comment:
Can you elaborate why this option is removed?

Reply:
Will restore it.